prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE >
+<html>
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
```

Comment:
you want to put this console log into the `main.js` file.

That will allow the script tag on line 10 to run the code from that file